docker/web/Dockerfile

```diff
@@ -23,6 +23,13 @@ WORKDIR /cbioportal
 RUN mvn -DskipTests clean install
 
 FROM shipilev/openjdk-shenandoah:8
+
+ENV PORTAL_WEB_HOME=/cbioportal-webapp
+
 COPY --from=build /cbioportal/portal/target/cbioportal*.war /app.war
 COPY --from=build /cbioportal/portal/target/dependency/webapp-runner.jar /webapp-runner.jar
-CMD /usr/bin/java ${JAVA_OPTS} -jar /webapp-runner.jar ${WEBAPP_OPTS} /app.war
+
+RUN mkdir -p $PORTAL_WEB_HOME
+RUN cd $PORTAL_WEB_HOME && /root/jdk/bin/jar -xvf /app.war
+
+CMD ["/bin/sh", "-c", "/usr/bin/java ${JAVA_OPTS} -jar /webapp-runner.jar ${WEB_OPTS} ${PORTAL_WEB_HOME}"]
```

Comment:
@pvannierop there's a typo here in `WEB_OPTS`, should be `${WEBAPP_OPTS}`

Also double check the the environment parameters are still correctly passed this way. CMD with square brackets vs w/o brackets still confuses me 🙂

Reply:
For reference, [these](https://docs.docker.com/engine/reference/builder/#cmd) are the docs on the semantics of the different forms.

Reply:
As I understand it, the form without square brackets is effectively syntactic sugar for `CMD ["sh", "-c", x]`, and doesn't play well with `ENTRYPOINT`. In both cases a shell is started to parse the dollar marks, but in the square bracket case it is explicit.

Reply:
WEB_OPTS was renamed to WEBAPP_OPTS.

Reply:
I renamed WEB_OPTS to WEBAPP_OPTS.